src/js/view/controls/nextuptooltip.js

```diff
@@ -154,7 +171,7 @@ define([
             const showUntilEnd = val >= this.offset;
             if (showUntilEnd && nextUpSticky === undefined) { // show if nextUpSticky is unset
                 this.nextUpSticky = showUntilEnd;
-                this.toggle(showUntilEnd);
+                this.toggle(showUntilEnd, 'time');
             } else if (!showUntilEnd && nextUpSticky === false) { // reset if there was a backward seek
```

Comment:
This should be `!showUntilEnd && nextUpSticky === true` so that we reset if nextUp should not be displayed because the position is before the nextUpOffset.